python/cudf/cudf/_lib/csv.pyx

```diff
@@ -554,7 +554,7 @@ cdef data_type _get_cudf_data_type_from_dtype(object dtype) except *:
     # TODO: Remove this work-around Dictionary types
     # in libcudf are fully mapped to categorical columns:
     # https://github.com/rapidsai/cudf/issues/3960
```

Comment:
This linked issue has been resolved. Is there an action to be taken here (i.e. removal of the workaround)?

Reply:
It appears although the issue is closed, the latest PR that addressed this function notes that it's still awaiting libcudf handling of category type: https://github.com/rapidsai/cudf/pull/12571#issue-1538618723